ComfortZoneApp/app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,14 @@
     <string name="create_account">Create account</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="welcome">Welcome!</string>
+    <string name="introduction">ComfortZone will track your comfort zones for different temperature levels! A \'0\' represents the point where you are crazy cold! Similarly, a \'10\' represents a crazy hot degree. And \'5\' is your estimated perfect temperature!</string>
+    <string name="what_is_your_0">What is your 0?</string>
+    <string name="what_is_your_5">What is your 5?</string>
+    <string name="what_is_your_10">What is your 10?</string>
+    <string name="sign_in">Sign in</string>
+    <string name="don_t_have_an_account">Don\'t have an account?</string>
+    <string name="signup">Signup</string>
+    <string name="hello">HELLO!</string>
+    <string name="your_perfect_temperature_is">Your perfect temperature is:</string>
```

Comment:
really nice!!

Reply:
Yes, good to see you incorporating feedback and making the changes. Great job Fiona :)